tests/integration/plots/test_plots.py

```diff
@@ -49,8 +49,7 @@ def extract_vega_specs(html_path, plots_ids):
     reader = BeautifulSoup(content, features="html.parser")
     for plot_id in plots_ids:
         clean_id = Renderer.remove_special_chars(plot_id)
-        div_id = f"plot_{clean_id}"
-        script = _remove_blanks(reader.find("div", id=div_id).script.text)
+        script = _remove_blanks(reader.find("div", id=clean_id).script.text)
```

Comment:
@pared There are some tests here marked as `vscode` but vscode doesn't rely on the generated HTML (afaik). Maybe we should separate to only mark with `vscode` the relevant logic?